h2/src/main/org/h2/mode/FunctionsPostgreSQL.java

```diff
@@ -101,6 +105,8 @@
                 PG_TABLE_IS_VISIBLE, 1, Value.BOOLEAN, true, false, true, false));
         FUNCTIONS.put("SET_CONFIG", new FunctionInfo("SET_CONFIG", //
                 SET_CONFIG, 3, Value.VARCHAR, true, false, true, false));
+        FUNCTIONS.put("ARRAY_TO_STRING", new FunctionInfo("ARRAY_TO_STRING", //
+                ARRAY_TO_STRING, 2, Value.VARCHAR, true, false, true, false));
```

Comment:
I think this function is expected to be deterministic and in the PostgreSQL it has variable number of arguments.

Reply:
As of PostgreSQL 8.2 this function only accepts 2 arguments (https://www.postgresql.org/docs/8.2/functions-array.html). The 3rd arg is available since PostgreSQL 9.1.

Reply:
It should use `false`, `true`, `true`, `false` because it doesn't return `NULL` on **any** `NULL` parameter and because it is deterministic.

Reply:
fixed